pom.xml

```diff
@@ -176,29 +190,33 @@
 		<dependency>
 			<groupId>org.mongodb</groupId>
 			<artifactId>mongodb-driver-sync</artifactId>
-			<scope>test</scope>
```

Comment:
Clarification Q. Why did we need this outside of the test scope?

Reply:
@tlawrie  At line 212 there was a duplicate entry, without <scope>, so I thought maybe someone needed it in 'live'.